cross/firebird/Makefile

```diff
@@ -0,0 +1,121 @@
+PKG_NAME = firebird
+PKG_VERS = 4.0.3
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/FirebirdSQL/firebird/archive/refs/tags
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
```

Comment:
please add PKG_DIST_FILE to rename downloaded file to make it unique (add package name)
after this, you have to recreate the digests file.
```suggestion
PKG_DIST_SITE = https://github.com/FirebirdSQL/firebird/archive/refs/tags
PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
```

Reply:
Done. I will push the other fixes in following days.